### DISCUSSION TITLE
DATABASE_URI is hardcoded somewhere

### DISCUSSION THREAD

**Discussion by mryellow** [payloadcms/payload#6703] [Q&A] [answered]
### Describe the Bug

Changing `DATABASE_URI` to `PAYLOAD_DATABASE_URI` 

Then providing it to `db: mongooseAdapter({ url: process.env.PAYLOAD_DATABASE_URI }),`.

Results in the string not being used but instead `DATABASE_URI` *MUST* be provided under that name.

### To Reproduce

Set `PAYLOAD_DATABASE_URI`  and pass to `mongooseAdapter`.

### Payload Version

2

**Comment by denolfe** [comment]
I was able to swap these values in the config and .env without issue. Could you provide a repo that reproduces this issue?

**Comment by mryellow** [comment]
Maybe it's when using the dev version of the admin UI. Seeing I was unable to run `build:payload`.

Perhaps in that situation  it uses a payload which has `DATABASE_URI` coded somewhere?

**Comment by jmikrut** [comment]
There is no `DATABASE_URI` hard-coded in the admin UI. We'll need more information about what you're trying to accomplish, and which errors you're seeing, with a reproducable repository in order to help here.

In the meantime, I will convert this to a discussion and we can continue to help there. If you do end up creating a repro, we'd be happy to look at it!

**Reply by mryellow** [reply]
I really dislike having reported issues swept aside like this.

**Reply by mryellow** [reply]
> We'll need more information about what you're trying to accomplish, and which errors you're seeing, with a reproducable repository in order to help here.

Then leave the issue open and ask for the details.

> If you do end up creating a repro, we'd be happy to look at it!

Having closed the issue. I am unlikely to be motivated to contribute further.

**Reply by jmikrut** [reply]
We try and reserve GitHub issues on the Payload repo to only contain reproducible bugs so that our team can focus our efforts. Converting to a discussion is not meant to be sweeping it aside - rather, I just put this question where it belongs as it was not reproducible and I am quite certain that there is no hard-coded reference to `DATABASE_URI`. 

We have lots of projects that do not use this environment variable naming convention and they all work fine.

To help here, I need to further understand what you're trying to do, and from there I would be happy to continue looking into this. 

If this is indeed an issue and we can reproduce it, then I will happily keep it open as a GitHub issue until it's resolved!

**Reply by mryellow** [reply]
> Converting to a discussion is not meant to be sweeping it asid

I won't be digging any deeper without an issue to report against.

**Comment by mryellow** [comment, marked as answer]
The `DATABASE_URI` is hard-coded in the dev version of the admin. Providing a new URI to the adaptor does not function.

This is likely a bug.
